### PR TITLE
adding yang formatting to compiler

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: pkg/compiler/compiler.go [hunk 1]
```diff
@@ -188,16 +171,6 @@ func (c *ModelCompiler) loadModelMetaData(path string) error {
 	return nil
 }
 
-func (c *ModelCompiler) loadPluginVersion(path string) error {
```
Comment (Andrea-Campanella) [inline pkg/compiler/compiler.go:191]: why was this removed ?

File: pkg/compiler/compiler.go [hunk 2]
```diff
@@ -220,6 +193,50 @@ func (c *ModelCompiler) lintModel(path string) error {
...
+	yangDir := filepath.Join(path, "yang")
+	yangDirs := []string{yangBaseDirectory, yangDir}
+
+	tempFile := fmt.Sprintf("%s/temp-formatted.yang", os.TempDir())
+	err := filepath.Walk(yangDir,
+		func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if info.IsDir() || !strings.HasSuffix(info.Name(), ".yang") {
```
Comment (Andrea-Campanella) [inline pkg/compiler/compiler.go:209]: do you want to extract the `yang` as a constant ?

**Review by Andrea-Campanella — COMMENTED** [review]
minor comments, otherwise LGTM

File: pkg/compiler/compiler.go (hunk 1 above)
Comment (SeanCondon) [reply pkg/compiler/compiler.go:191]: because it was not being used in any template - Charles had removed the plugin `VERSION` file in the sd-fabric models, and this was throwing an error - I started investigating where is was used and it's not
